### PR TITLE
DCA-461 refactor analytics and utils

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

File: app/util/analytics/analytics.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+               "os": collector.os,
+               "tool_ver": collector.tool_version,
+               "exp_dur": collector.duration,
+               "act_dur": collector.actual_duration,
+               "selenium_test_rates": collector.selenium_test_rates,
+               "jmeter_test_rates": collector.jmeter_test_rates,
+               "concurrency": collector.concurrency
+               }
+    r = requests.post(url=f'{BASE_URL}', json=payload, headers=headers)
+    print(r.json())
```
Comment (mmizin) [inline app/util/analytics/analytics.py:119]: Are You printing 'r' to check the result and You just forgot to delete it or it is needed print?

Comment (SergeyMoroz0703) [reply app/util/analytics/analytics.py:119]: Excellent catch!

File: app/util/analytics/analytics.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    if r.status_code != 200:
+        print(f'Analytics data was send unsuccessfully, status code {r.status_code}')
+
+
+def main():
+    application_name = get_first_elem(sys.argv)
+    application = ApplicationSelector(application_name).application
+    collector = AnalyticsCollector(application)
+    generate_report_summary(collector)
+    # if collector.is_analytics_enabled():
```
Comment (ometelytsia) [inline app/util/analytics/analytics.py:138]: do not forget to uncomment before merge

File: app/util/analytics/analytics_utils.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        summary_report.append(f'Total Git operations count|{total_git_count}')
+        summary_report.append(f'Total Git operations compliant|{git_compliant}')
+
+    summary_report.append(f'Finished|{finished}')
+    summary_report.append(f'Compliant|{compliant}')
+    summary_report.append(f'Success|{success}\n')
+
+    summary_report.append(f'Action|Success Rate|Status')
+    load_test_rates = collector.jmeter_test_rates if collector.conf.load_executor == 'jmeter' \
+        else collector.locust_test_rates
```
Comment (ometelytsia) [inline app/util/analytics/analytics_utils.py:61]: it would be nice to check executor types explicitly

**Comment by ometelytsia** [comment]
Need to check if locust.out could be removed by `app/util/cleanup_results_dir.py`

File: app/util/analytics/analytics.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+               "app_version": collector.application_version,
+               "date": collector.date,
+               "time_stamp": collector.time_stamp,
+               "app_type": collector.app_type,
+               "os": collector.os,
+               "tool_ver": collector.tool_version,
+               "exp_dur": collector.duration,
+               "act_dur": collector.actual_duration,
+               "selenium_test_rates": collector.selenium_test_rates,
+               "jmeter_test_rates": collector.jmeter_test_rates,
```
Comment (ometelytsia) [inline app/util/analytics/analytics.py:124]: need to add locust_test_rates

**Pull request merged**